client/components/GLogout.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { GoogleLogout } from 'react-google-login'
+import getConfig from "next/config";
+
+const { publicRuntimeConfig } = getConfig();
+
+
+function Logout() {
```

Comment:
The same as `Login` component.
```suggestion
function GLogout() {
```

Also, you can pass the `onSuccess` event handler to the user since there is no specific usage in this component.
```suggestion
function GLogout({onSuccess}) {
```